Hyper/Scene.cs

```diff
@@ -308,9 +240,18 @@ public void RegisterCallbacks()
             {
                 movementDirection += new Vector2(1, 0);
             }
-            _character.UpdateCharacterGoals(_simulationManager.Simulation, Camera, (float)e.Time,
+            _player.UpdateCharacterGoals(_simulationManager.Simulation, Camera, (float)e.Time,
                 tryJump: context.HeldKeys[Keys.Space], sprint: context.HeldKeys[Keys.LeftShift],
-                Conversions.ToNumericsVector(movementDirection));
+                movementDirection);
+            foreach (var bot in _bots)
+            {
+                float tMs = _stopwatch.ElapsedMilliseconds;
```

Comment:
`e.Time` has time elapsed since previous render frame, I needed total elapsed time for that.
This is just an illustration of how the bots should be updated, we'll need smarter bots in the future anyway, so I'd leave it as it is for now.